app/diagnostics.go

```diff
@@ -330,6 +331,13 @@ func trackConfig() {
 		"show_full_name":     utils.Cfg.PrivacySettings.ShowFullName,
 	})
 
+	SendDiagnostic(TRACK_CONFIG_THEME, map[string]interface{}{
+		"enable_theme_selection": *utils.Cfg.ThemeSettings.EnableThemeSelection,
+		"default_theme":          *utils.Cfg.ThemeSettings.DefaultTheme,
```

Comment:
@jasonblais this is a user defined string. Should it be reported, or should it just report whether the default value has been changed or not?

Reply:
@dmeza can you help update this one to use the `isdefault()` function, similar to https://github.com/mattermost/platform/blob/master/app/diagnostics.go#L205?

That way it'll check if the default is used, but not actually take the user defined string.

Reply:
@grundleborg @jasonblais: the user defined string is for "default_theme"?

Reply:
Ah yes, that's correct. For the `DefaultTheme` setting, I think just knowing if the default was used is enough.

All other settings are fine as-is